tests/test_data_files.txt

```diff
@@ -0,0 +1,3 @@
+http://cdat.llnl.gov/cdat/sample_data
+aa51aa896ead44019335cebec68e7955  clt.nc
+9f6d62ea4393522dccfa64e092d8af91  sampleCurveGrid4.nc
```

Comment:
Are we testing that the files are actually being downloaded?

Reply:
yes, I am adding testcases.
I am making more changes..sorry, trying to add more testcases.
so I will wait before merging. thanks.